src/PrintInitDialog.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
+
+<html>
+
+<!-- Please note that Qt's TextBrowser renders things slightly unusually, so
+ a normal browser alone is not a sufficient test. Just edit this file and rerun
+ make to do your testing.  -->
+
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+
+  <style type=text/css>
+    body {
+      font: 13pt Arial,sans-serif;
+    }
+    .flattr-link {
+      color: white;
+      background: #88B058;
+      font: bold 14px Helvetica,sans-serif;
+    }
+    .donate-link {
+      color: white;
+      background: dodgerblue;
+      font: bold 14px Helvetica,sans-serif;
+    }
+  </style>
+</head>
+
+<body>
+
+<h1>Exporting Designs for 3D Printing</h1>
+
+<p>This feature allows direct upload of the current design to a 3D Printing Service.</p>
+
+<h2>Print a Thing</h2>
```

Comment:
As briefly discussed, since the URL points at `print.openscad.org`, the service in questions depends on who manages that backend server, which isn't tied to the OpenSCAD binary. We should probably pull this info from our website.
..and if we're offline, we cannot use it anyway, so perhaps we need a check for that?

Reply:
Sure, read info via network and if not available fall back to just local OctoPrint.